System.IO.Abstractions.TestingHelpers.Tests/MockFileStreamTests.cs

```diff
@@ -29,7 +29,7 @@ public void MockFileStream_Flush_WritesByteToFile()
         public void MockFileStream_Dispose_ShouldNotResurrectFile()
         {
             var fileSystem = new MockFileSystem();
-            var path = XFS.Path("C:\\test");
+            var path = XFS.Path("C:\\some_folder\\test");
```

Comment:
Why is this change necessary?

Reply:
Because of potential bug. Consider this code:
```
            var fileSystem = new MockFileSystem();
            var path = XFS.Path("C:\\test.txt");
            DirectoryInfoBase dir = fileSystem.Directory.GetParent(path);
```
It works ok on Windows but throws NullReferenceException on Linux. And after my changes GetParent() method is called inside File.Delete(), so this test started to fail. Adding a new intermediate folder "some_folder" is the fastest way to make the test work again.

I can create a new issue with this if you think it is a real defect.

Reply:
Ah OK, thanks. Could you please create an issue for this problem and add a comment to the changed test case that refers to the issue?

Reply:
Thanks!